### PR TITLE
Random mappings

### PR DESCRIPTION
**Pull request by anonymous123-code**
- World properties
- Explosions
- Projectiles
- Breeze
- wind charge
- world properties

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/Entity.mapping:662]: #553 calls this `getProjectileDeflector`, I can't check source rn to see which I prefer

File: mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping:1]: #553 calls this `ProjectileDeflector`

File: mappings/net/minecraft/world/explosion/Explosion.mapping
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/explosion/Explosion.mapping:75]: `CausingEntity` is a little awkward, would `Owner` 0r `Igniter` make sense?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping
Comment (supersaiyansubtlety) [reply mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping:1]: Given it has a `deflect` method, `ProjectileDeflector` seems more appropriate (but can't check source rn)

**Comment by anonymous123-code** [comment]
This PR duplicates some stuff done in #553, Ill wait until that one gets merged before updating this

**Comment by eristhea** [comment]
@anonymous123-code #553 has been merged, feel free to finish up this PR!

**Comment by eristhea** [comment]
mostly superceded by various PRs

**Pull request closed without merging**
